package.json

```diff
@@ -1,12 +1,227 @@
+    "lodash.truncate": "^4.4.2",
+    "longest-streak": "^2.0.4",
+    "map-obj": "^4.2.1",
+    "mathml-tag-names": "^2.1.3",
+    "mdast-util-from-markdown": "^0.8.5",
+    "mdast-util-to-markdown": "^0.6.5",
+    "mdast-util-to-string": "^2.0.0",
+    "meow": "^9.0.0",
+    "micromark": "^2.11.4",
+    "min-indent": "^1.0.1",
+    "minimist-options": "^4.1.0",
+    "natural-compare": "^1.4.0",
+    "normalize-package-data": "^3.0.2",
+    "normalize-range": "^0.1.2",
+    "normalize-selector": "^0.2.0",
+    "num2fraction": "^1.2.2",
+    "object-inspect": "^1.11.0",
+    "object-keys": "^1.1.1",
+    "object.assign": "^4.1.2",
+    "object.entries": "^1.1.4",
+    "object.values": "^1.1.4",
+    "optionator": "^0.9.1",
+    "parent-module": "^1.0.1",
+    "parse-entities": "^2.0.0",
+    "parse-json": "^5.2.0",
+    "path-key": "^3.1.1",
+    "path-parse": "^1.0.7",
+    "pify": "^3.0.0",
+    "pkg-dir": "^2.0.0",
+    "pkg-up": "^2.0.0",
+    "postcss-html": "^0.36.0",
+    "postcss-media-query-parser": "^0.2.3",
+    "postcss-resolve-nested-selector": "^0.1.1",
+    "postcss-syntax": "^0.36.2",
+    "prelude-ls": "^1.2.1",
+    "progress": "^2.0.3",
+    "read-pkg": "^5.2.0",
+    "read-pkg-up": "^7.0.1",
+    "redent": "^3.0.0",
+    "regexpp": "^3.2.0",
+    "remark": "^13.0.0",
+    "remark-parse": "^9.0.0",
+    "remark-stringify": "^9.0.1",
+    "repeat-string": "^1.6.1",
+    "resolve": "^1.20.0",
+    "resolve-from": "^5.0.0",
+    "shebang-command": "^2.0.0",
+    "shebang-regex": "^3.0.0",
+    "slice-ansi": "^4.0.0",
+    "spdx-correct": "^3.1.1",
+    "spdx-exceptions": "^2.3.0",
+    "spdx-expression-parse": "^3.0.1",
+    "spdx-license-ids": "^3.0.9",
+    "specificity": "^0.4.1",
+    "sprintf-js": "^1.0.3",
+    "string.prototype.trimend": "^1.0.4",
+    "string.prototype.trimstart": "^1.0.4",
+    "strip-bom": "^3.0.0",
+    "strip-indent": "^3.0.0",
+    "strip-json-comments": "^3.1.1",
+    "style-search": "^0.1.0",
+    "stylelint-config-recommended": "^4.0.0",
+    "sugarss": "^2.0.0",
+    "svg-tags": "^1.0.0",
+    "table": "^6.7.1",
+    "text-table": "^0.2.0",
+    "to-fast-properties": "^2.0.0",
+    "trim-newlines": "^3.0.1",
+    "trough": "^1.0.5",
+    "tsconfig-paths": "^3.10.1",
+    "type-check": "^0.4.0",
+    "type-fest": "^0.20.2",
+    "unbox-primitive": "^1.0.1",
+    "unified": "^9.2.2",
+    "unist-util-find-all-after": "^3.0.2",
+    "unist-util-is": "^4.1.0",
+    "unist-util-stringify-position": "^2.0.3",
+    "v8-compile-cache": "^2.3.0",
+    "validate-npm-package-license": "^3.0.4",
+    "vfile": "^4.2.1",
+    "vfile-message": "^2.0.4",
+    "which": "^2.0.2",
+    "which-boxed-primitive": "^1.0.2",
+    "word-wrap": "^1.2.3",
+    "yaml": "^1.10.2",
+    "yargs-parser": "^20.2.9",
+    "zwitch": "^1.0.5"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch",
+    "start": "webpack serve --open",
+    "build": "webpack"
```

Comment:
Well done implementing this but you want to add the mode for development and production in your script like this below 
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
```

Reply:
ok @abruzy thank you for the review